db/migrate/20200412194408_create_user_optional_fields.rb

```diff
@@ -0,0 +1,13 @@
+class CreateUserOptionalFields < ActiveRecord::Migration[5.2]
+  def change
+    create_table :user_optional_fields do |t|
+      t.string :label
+      t.string :value
+      t.references :user, foreign_key: true
+
+      t.timestamps
+    end
+
+    add_index :user_optional_fields, :label , unique: true
```

Comment:
We need the index to be unique for the two fields, not just for the label, same as in Rails as you correctly changed

```suggestion
    add_index :user_optional_fields, [:label, :user_id], unique: true
```

Reply:
Ah, thanks for explaining.

Done!